### PR TITLE
Event page: update speaker cards

### LINKED ISSUE #69: Headshots are not uniform across events
Dante's image appears in the lower left of the event panel, but not in the Canva-created social media card.

Sandra's image appears in the Canva-created social media card, but not in the lower left of the event panel.

image

### PR DESCRIPTION
**Pull request by reshamas**
Closes #69

### CODE REVIEW

**Comment by canyon289** [comment]
thank you for all these updates. One question regarding the images. Are they supposed to look like this or is the github svg tenderer making things look strange?

image

**Comment by reshamas** [comment]
Not sure, the previews look ok here:
https://deploy-preview-75--pymcon.netlify.app/events/

**Comment by reshamas** [comment]
@canyon289 
The SVG files look ok to me:
https://github.com/pymc-devs/pymcon_web_series_website/pull/75/files?file-filters%5B%5D=.svg&show-viewed-files=true

Are you checking on mobile?

**Comment by canyon289** [comment]
Huh..... Im on my macbook. In those same link I'm still seeing issues as well. Would it be safer to use jpgs or pngs to avoid local rendering issues like this one

image

image

image

**Review by canyon289 — APPROVED** [review]
Thanks much for these updates, this really fills out the details on the site. Everything is looking good on my end, save some weird TOML error that is breaking rendering.

Feel free to merge whenever ready

File: data/events.toml [hunk 1]
```diff
@@ -18,75 +18,70 @@
...
 
 [[event]]
-    dataTime = "Feb 18th"
+    dataTime = "Feb 18 2023"
     title = "An Introduction To Multi-Output Gaussian Processes Using PyMC"
     place = "Online"
     description = "The talk aims to get users quickly up and performing GPs, especially multi-output GPs using PyMC. Several examples with time-series datasets are used to illustrate different GPs features. This presentation will allow users to leverage GPs to analyze their data effectively."
-
-    bannerImgPath = "/images/events/pymcon-Danh.png"
+    bannerImgPath = "/images/events/pymcon-danh-phan.png
```
Comment (canyon289) [inline data/events.toml:39]: I think the issue is here 
```suggestion
    bannerImgPath = "/images/events/pymcon-danh-phan.png"
```

**Comment by canyon289** [comment]
🎉 🎉

**Pull request merged**